SDWebImage/Core/SDAnimatedImage.m

```diff
@@ -375,7 +377,8 @@ - (SDImageFormat)sd_imageFormat {
 }
 
 - (void)setSd_imageFormat:(SDImageFormat)sd_imageFormat {
```

Comment:
This override method does nothing (The same as super implementation actually). You can just remove this code